aleph_alpha_client/image.py

```diff
@@ -93,6 +96,40 @@ def __init__(
         self.cropping = cropping
         self.controls: Sequence[ImageControl] = controls
 
+    @classmethod
+    def from_image_source(
+        cls,
+        image_source: Union[str, Path, bytes],
+        controls: Optional[Sequence[ImageControl]] = None,
+    ):
+        """
+        Abstraction on top of the existing methods of image initialization.
+        If you are not sure what the exact type of your image, but you know it is either a URL, a file path,
+        or a bytes array, just use the method and we will figure out which of the methods of image initialization to use
+        The image will be [center cropped](https://pytorch.org/vision/stable/transforms.html#torchvision.transforms.CenterCrop)

```

Comment:
Probably better to move this comment to the class' docstring

Reply:
I'm not sure what you mean here. Could you please elaborate?

Reply:
@volkerstampa  I'm still not 100% sure, what you mean? Could you please write the comment and I will include it in the update?